src/evidently/pipeline/column_mapping.py

```diff
@@ -27,6 +27,7 @@ class ColumnMapping:
     task: Optional[str] = None
     pos_label: Optional[Union[str, int]] = 1
     text_features: Optional[List[str]] = None
+    embeddings: Optional[Dict[str, List[str]]] = None
```

Comment:
You can create an alias for this type, eg `Embeddings = Dict[str, List[str]]` so it's clearer to use